src/test/java/ru/netology/manager/AfishaManagerTest.java

```diff
@@ -9,35 +9,53 @@
 
 class AfishaManagerTest {
     private AfishaManager manager = new AfishaManager();
-    private Movie first = new Movie(1, "first", "Мультфильм", "www.1ссылка.com");
-    private Movie second = new Movie(2, "second", "Ужасы", "www.2ссылка.com");
-    private Movie third = new Movie(3, "third", "Боевик", "www.3ссылка.com");
-    private Movie fourth = new Movie(4, "fourth", "Мюзикал", "www.4ссылка.com");
-    private Movie fifth = new Movie(5, "fifth", "Драма", "www.5Url.com");
-    private Movie sixth = new Movie(6, "sixth", "Документалка", "www.6Url.com");
-    private Movie seventh = new Movie(7, "seventh", "Детектив", "www.7Url.com");
-    private Movie eighth = new Movie(8, "eighth", "Комедия", "www.8Url.com");
-    private Movie ninth = new Movie(9, "ninth", "ХХХ", "www.9Url.com");
-    private Movie tenth = new Movie(10, "tenth", "Шляпа", "www.10Url.com");
-
-    @BeforeEach
+    private final Movie first = new Movie(1, "first", "Мультфильм");
+    private final Movie second = new Movie(2, "second", "Ужасы");
+    private final Movie third = new Movie(3, "third", "Боевик");
+    private final Movie fourth = new Movie(4, "fourth", "Мюзикал");
+    private final Movie fifth = new Movie(5, "fifth", "Драма");
+
     public void setUp() {
         manager.add(first);
         manager.add(second);
         manager.add(third);
         manager.add(fourth);
         manager.add(fifth);
-        manager.add(sixth);
-        manager.add(seventh);
-        manager.add(eighth);
-        manager.add(ninth);
     }
 
     @Test
-    void getLastWithQuantityMoreThanInList() {
-        Movie[] actual = manager.getLast();
-        Movie[] expected = new Movie[]{ninth, eighth, seventh, sixth, fifth, fourth, third, second, first};
+    public void addMoviesToEmpty() {
+        manager.add(first);
+        assertArrayEquals(new Movie[]{first}, manager.getAll());
+    }
+
+    @Test
+    public void addMoviesToExisted() {
+        manager.add(first);
+        Movie[] expected = new Movie[]{first};
+        Movie[] actual = manager.getAll();
+        assertArrayEquals(new Movie[]{first}, manager.getAll());
+
+        manager.add(second);
+        Movie[] expectedAdded = new Movie[]{second, first};
+        Movie[] actualAdded = manager.getAll();
+        assertArrayEquals(expectedAdded, actualAdded);
+    }
+
+    @Test
+    void moviesForFeed() {
+        setUp();
+        Movie[] expected = new Movie[]{fifth, fourth, third, second, first};
+        Movie[] actual = manager.getMovies();
         assertArrayEquals(expected, actual);
     }
 
-}
+    @Test
+    void MoviesMinFeed() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.